### PR TITLE
Refactor many things

### PR DESCRIPTION
**Pull request by SollyW**
I'll pr this to make the changes visible. They are not yet complete.

### CODE REVIEW

**Comment by SollyW** [comment]
Tf are you on about ci

**Comment by CDAGaming** [comment]
```
/home/runner/work/The-Aether/The-Aether/src/main/java/com/aether/items/tools/SkyrootBucketItem.java:179: error: cannot find symbol
                    this.CPAonFluidPlaced(worldIn, posIn); //trigger CPA to attempt a portal light.
                        ^
  symbol: method CPAonFluidPlaced(World,BlockPos)
  ```
  onFluidPlaced would be the correct term here afaik, as I dont know anyone who would break camelCase standards.

**Comment by CDAGaming** [comment]
Also if you wanted to, you could do your accessWidener reduction changes in here, as you mentioned awhile back that most could be replaced.

**Comment by SollyW** [comment]
> ```
> /home/runner/work/The-Aether/The-Aether/src/main/java/com/aether/items/tools/SkyrootBucketItem.java:179: error: cannot find symbol
>                     this.CPAonFluidPlaced(worldIn, posIn); //trigger CPA to attempt a portal light.
>                         ^
>   symbol: method CPAonFluidPlaced(World,BlockPos)
> ```
> 
> onFluidPlaced would be the correct term here afaik, as I dont know anyone who would break camelCase standards.

I didn't write it; I'm not sure what's wrong there

**Comment by CDAGaming** [comment]
@SollyW Rebase on newer changes to patch CI issues, mainly so we can have this have a workable CI build to compare with.

**Comment by Jack-Papel** [comment]
Not sure if you saw the discord message, but we also agreed upon registering block items in the items class.

File: src/main/java/com/aether/mixin/entity/BoatEntityMixin.java [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-        switch (((BoatEntity) (Object) this).getBoatType().getName()) {
-            case "aether_skyroot" -> cir.setReturnValue(AetherItems.SKYROOT_BOAT);
-            case "aether_golden_oak" -> cir.setReturnValue(AetherItems.GOLDEN_OAK_BOAT);
-            case "aether_orange" -> cir.setReturnValue(AetherItems.ORANGE_BOAT);
-            case "aether_crystal" -> cir.setReturnValue(AetherItems.CRYSTAL_BOAT);
-            case "aether_wisteria" -> cir.setReturnValue(AetherItems.WISTERIA_BOAT);
-        }
+        BoatEntity.Type type = this.getBoatType();
+        if (type == AetherBoatTypes.SKYROOT) cir.setReturnValue(AetherItems.SKYROOT_BOAT);
+        else if (type == GOLDEN_OAK) cir.setReturnValue(AetherItems.GOLDEN_OAK_BOAT);
```
Comment (Jack-Papel) [inline src/main/java/com/aether/mixin/entity/BoatEntityMixin.java]: Why is Golden Oak imported unlike the others?

File: src/main/java/com/aether/blocks/AetherBlocks.java [hunk 2]
```diff
@@ -1,680 +1,456 @@
...
+
+    private static Settings unbreakable(Settings settings) { return settings.strength(-1f, 3600000f); }
+    private static Settings flowerPot() { return copy(POTTED_OAK_SAPLING); }
+    
+    private static Action<Block> flammable(int spread, int burn) { return (id, block) -> ((FireBlock) FIRE).registerFlammableBlock(block, spread, burn); }
+    private static final Action<Block> flammableLog = flammable(5, 5);
+    private static final Action<Block> flammablePlanks = flammable(20, 5);
+    private static final Action<Block> flammableLeaves = flammable(60, 30);
+    private static final Action<Block> flammablePlant = flammable(60, 100);
+    private static final Action<Block> translucenRenderLayer = onClient((id, block) -> RenderLayers.BLOCKS.put(block, RenderLayer.getTranslucent()));
```
Comment (Jack-Papel) [inline src/main/java/com/aether/blocks/AetherBlocks.java]: `translucen` -> `translucent`

File: src/main/java/com/aether/entities/block/FloatingBlockEntity.java [hunk 3]
```diff
@@ -210,6 +204,7 @@ public void postTickEntities() {
 
                     if (blockHitResult.getType() != HitResult.Type.MISS
                             && this.world.getFluidState(blockHitResult.getBlockPos()).isIn(FluidTags.WATER)) {
+
```
Comment (Jack-Papel) [inline src/main/java/com/aether/entities/block/FloatingBlockEntity.java]: Why is there an extra line here?

**Review by Jack-Papel — CHANGES_REQUESTED** [review]
Food bowls no longer update their texture when food is put in them.
They also sometimes can't be placed

**Comment by SollyW** [comment]
I can't reproduce an inability to place the food bowls, but the rendering has been fixed

**Review by Jack-Papel — APPROVED** [review]
Looks good to me 👍🏻

**Review by Azzyypaaras — APPROVED** [review]
bless_2

**Pull request merged**